### ISSUE TITLE
DllNotFoundException

### ISSUE THREAD

**Issue by andreobvio** [ladenedge/WebRtcVadSharp#12]
Hello, I am using webrtcvadsharp inside Unity, Ive imported that using NuGet.

<img width="691" alt="Screenshot 2023-01-20 at 16 03 39" src="https://user-images.githubusercontent.com/114940629/213784390-b0f71d10-72d9-4f95-bda1-dcd06d3060e1.png">

full log:
DllNotFoundException: webrtcvad assembly:<unknown assembly> type:<unknown type> member:(null)
WebRtcVadSharp.WebRtc.WebRtcDll.Create () (at <d218574332e64c9792a446c6d48f7bc0>:0)
WebRtcVadSharp.WebRtcVad..ctor (WebRtcVadSharp.WebRtc.IWebRtcDll library) (at <d218574332e64c9792a446c6d48f7bc0>:0)
Rethrow as DllNotFoundException: Unable to load DLL 'WebRtcVad.dll' or a dependency. Be sure it exists in '/Users/andrescapinlli/Documents/Baires Dev/ObvioHealth/ObvioSpeech/obviospeech' or elsewhere in the DLL search path.
WebRtcVadSharp.WebRtcVad..ctor (WebRtcVadSharp.WebRtc.IWebRtcDll library) (at <d218574332e64c9792a446c6d48f7bc0>:0)
WebRtcVadSharp.WebRtcVad..ctor () (at <d218574332e64c9792a446c6d48f7bc0>:0)
MicrophoneController.DoesFrameContainSpeech (System.Byte[] audioFrame) (at Assets/Scripts/Recorder/MicrophoneController.cs:56)
MicrophoneController.Update () (at Assets/Scripts/Recorder/MicrophoneController.cs:51)

-----------------------------------------------------------------------------------------

So basically my code is not finding the .dll. Does someone have any idea how to solve that?

I`ve already tried to change the path to the root of my project, but it also does not work.

**Comment by tonyabracadabra** [comment]
Hey Jay @ladenedge , I am facing the same issue, do you have any advice on it? There aren't really a lot of other VAD libraries in Unity out there and your repo can really be a life saver to me! Thanks a ton!

**Comment by zhouchanglin-rr** [comment]
We have two options:
1) use Visual studio to build WebRtcVAD.dll
2) use mingw toolchain to build
The best solution is to provide an official native dll in the nuget package.
